cmd/rokka/main.go

```diff
@@ -46,5 +48,29 @@ func main() {
 		APIKey: cfg.APIKey,
 	})
 
-	cli.ExecCommand(cl, args)
+	err = cli.ExecCommand(cl, args)
+
+	if err == nil {
+		os.Exit(0)
+	}
+
+	switch err.(type) {
+	case cli.UnknownCommandError:
+		fmt.Fprintf(os.Stderr, "Error: %v\n\n", err)
+		fmt.Fprint(os.Stderr, "Commands:\n")
+		printCommands(os.Stderr)
+	case rokka.StatusCodeError:
+		fmt.Fprintf(os.Stderr, "Error: %v\n\n", err)
+		fmt.Fprintf(os.Stderr, "%s\n", cli.PrettyJSON(err.(rokka.StatusCodeError).Body))
+	default:
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+	}
+
+	os.Exit(1)
```

Comment:
should we use different exit codes depending on the error?

Reply:
Wouldn't do that at this point.